.circleci/config.yml

```diff
@@ -1,15 +1,12 @@
 version: 2.1
 
 executors:
-  py38:
-    docker:
-      - image: cimg/python:3.8
-  py39:
-    docker:
-      - image: cimg/python:3.9
   py310:
     docker:
       - image: cimg/python:3.10
+  py312:
+    docker:
+      - image: cimg/python:3.12
```

Comment:
Everything we care about/where we consume this package should be running python 3.10 or 3.12 now.
We don't need to keep testing for python 3.8/3.9 compatibility.